recipes/godot_headers/all/conanfile.py

```diff
@@ -0,0 +1,27 @@
+from conans import ConanFile
+from conans import tools
+
+
+class GodotHeadersConan(ConanFile):
+    name = "godot_headers"
```

Comment:
It's a personal preference, but doesn't `godot-headers` sound better?

Reply:
I kept it as godot_headers since that is what the repo/project is called upstream.

Reply:
It's also in line with your godot-cpp recipe.

Reply:
Don't mind this comment then.